Lib/platform.py

```diff
@@ -315,18 +315,24 @@ def _syscmd_ver(system='', release='', version='',
 try:
     import _wmi
 except ImportError:
-    def _wmi_query(*keys):
+    def _wmi_query(table, *keys):
         raise OSError("not supported")
 else:
     def _wmi_query(table, *keys):
         table = {
             "OS": "Win32_OperatingSystem",
             "CPU": "Win32_Processor",
         }[table]
-        data = _wmi.exec_query("SELECT {} FROM {}".format(
-            ",".join(keys),
-            table,
-        )).split("\0")
+        try:
+            data = _wmi.exec_query("SELECT {} FROM {}".format(
+                ",".join(keys),
+                table,
+            )).split("\0")
+        except OSError:
+            global _wmi_query
+            def _wmi_query(table, *keys):
+                raise OSError("not supported")
+            raise OSError("not supported")
```

Comment:
Let's restructure this import block to make `_wmi = None` when it fails to import, and then if we fail to query it can clear out `_wmi` and subsequent calls will act as if it were never imported.